src/OrchardCore.Modules/OrchardCore.Media/Settings/MediaFieldSettings.cs

```diff
@@ -9,6 +9,8 @@ public class MediaFieldSettings
 
         [DefaultValue(true)]
         public bool Multiple { get; set; } = true;
+        
+        [DefaultValue(true)]
```

Comment:
This is the first time I saw this attribute, why it's useful, while we assign a value to the property itself?

Reply:
It's required by the serializer for content type / part settings.

https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_DefaultValueHandling.htm